evento/src/test-helpers/mock-factory.js

```diff
@@ -6,6 +6,13 @@ const generateId = () =>
 const generateString = (n) =>
 	(Math.random().toString(36) + '00000000000000000').slice(2, n + 2);
 
+// Return time stamp of day after tomorrow
+const generateTimeStamp = () => {
```

Comment:
I'm not sure if that's relevant. Only requirement really is that it's valid. Like generateString doesn't really matter what the string is as long it is valid.